cmd/earthly/build_cmd.go

```diff
@@ -270,7 +268,7 @@ func (app *earthlyApp) actionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs
 	debuggerSettings := debuggercommon.DebuggerSettings{
 		DebugLevelLogging: app.debug,
 		Enabled:           app.interactiveDebugging,
-		RepeaterAddr:      fmt.Sprintf("%s:8373", bkIP),
+		SocketPath:        "/var/run/earthly_interactive", // TODO const
```

Comment:
This is a path on the host system, right?

Reply:
it's actually under the container; I forgot about my TODO, hopefully it's more clear now.